recipe/meta.yaml

```diff
@@ -1,5 +1,5 @@
 {% set build_num = 2 %}
-{% set gcc_major = 13 if ctng_gcc is undefined else ctng_gcc.split(".")[0] %}
+{% set gcc_major = 13 if compiler_version is undefined else compiler_version.split(".")[0] %}
```

Comment:
This value will be 16 or 17 for clang?

Reply:
That's true, though all the outputs or lines that use `gcc_major` are skipped for clang builds. I'd be happy to add another condition on `{% if compiler_vendor == "gcc" %}` if you'd like.

Reply:
`min_runtime_version` is a derived from `gcc_major` and is used as a `run_exports` in `clang`.

Reply:
I changed the run-export to use `last_symbol_bump` (==12). It seemed to me like a reasonable base GCC version - we don't need the newest GCC when using clang, and we have to use the `last_symbol_bump` as the run-export lower bound in any case for anything below.